config/config.go

```diff
@@ -61,16 +70,18 @@ type DbConfig struct {
 	// PreparedDbFilePath is the path to the prepared database file.
 	PreparedDbFilePath string `mapstructure:"prepared_db_file_path"`
 
-	Postgres PostgreSqlConfig `mapstructure:"postgres"`
+	Postgres PostgresqlConfig `mapstructure:"postgres"`
```

Comment:
should not be `PostgreSQLConfig` ?

Reply:
go is recommending to use  capital letters for acronyms

Reply:
<img width="459" alt="image" src="https://github.com/bitcoin-sv/block-headers-service/assets/17731230/df6cb046-82a2-42a1-b591-4324c3400dbf">

Reply:
That's why I change it this way
But I see that they are not consistent about this name, sometimes they have Postgresql and sometimes it is PostgreSQL. If you don't mind, I would leave it as it is, because I don't want to go again and change every Posgresql into PostgreSQL

Reply:
well, it not consistent and the name contain SQL up to u ... i posted what i think

Reply:
Ok, I will change it, because the group behind it is `The PostgreSQL Global Development Group` so seems that SQL is more correct.